forui/lib/src/widgets/dialog/dialog_content.dart

```diff
@@ -0,0 +1,178 @@
+part of 'dialog.dart';
+
+@internal sealed class FDialogContent extends StatelessWidget {
+  final FDialogContentStyle style;
+  final CrossAxisAlignment alignment;
+  final String? title;
+  final TextAlign titleTextAlign;
+  final String? subtitle;
+  final TextAlign subtitleTextAlign;
```

Comment:
Feels a bit weird to offer `alignment` and `titleTextAlign`/`subtitleTextAlign`. Are there cases where we would like to use all three of the fields?

Reply:
These aren't exposed directly in the style. We expose these 3 fields since the alignment of the title/subtitle is towards the left in a horizontal layout, and centered in a vertical layout.